examples/envoy/README.md

```diff
@@ -0,0 +1,63 @@
+# Envoy SDS Example
+
+## Build
```

Comment:
It looks like Golang is a requirement for this example? This may have not been a problem when it previously lived in SPIRE repo, but now that it is standalone we should list this as a requirement

```suggestion
## Requirements

- Go 1.12
- docker-compose

## Build
```